plugins/org.python.pydev.parser/src/org/python/pydev/parser/jython/ast/Slice.java

```diff
@@ -63,6 +64,12 @@ public Slice createCopy(boolean copyComments) {
                 if(o instanceof commentType){
                     commentType commentType = (commentType) o;
                     temp.getSpecialsAfter().add(commentType.createCopy(copyComments));
+                } else if (o instanceof SpecialStr) {
```

Comment:
This should not be needed... also, if this was ever needed you'd have to change `asdl_java.py` to generate the code for you (the AST code is all generated).